loleaflet/src/control/Control.NotebookbarBuilder.js

```diff
@@ -625,6 +626,38 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 		}
 	},
 
+	_CharSpacing: function(parentContainer, data, builder) {
+		var options = {hasDropdownArrow: true};
+		var control = builder._unoToolButton(parentContainer, data, builder, options);
+
+		$(control.container).unbind('click');
+		$(control.container).click(function () {
+			var isChecked = function(command) {
+				var items = builder.map['stateChangeHandler'];
+				var val = items.getItemValue(command);
+				if (val && (val === 'true' || val === true))
+					return true;
+				else
+					return false;
+			};
+
+			$(control.container).w2menu({
+				items: [
+					{id: 'spacepara1', img: 'spacepara1', text: _UNO('.uno:SpacePara1'), uno: 'SpacePara1', checked: isChecked('.uno:SpacePara1')},
+					{id: 'spacepara15', img: 'spacepara15', text: _UNO('.uno:SpacePara15'), uno: 'SpacePara15', checked: isChecked('.uno:SpacePara15')},
+					{id: 'spacepara2', img: 'spacepara2', text: _UNO('.uno:SpacePara2'), uno: 'SpacePara2', checked: isChecked('.uno:SpacePara2')},
+					{type: 'break'},
+					{id: 'paraspaceincrease', img: 'paraspaceincrease', text: _UNO('.uno:ParaspaceIncrease'), uno: 'ParaspaceIncrease'},
+					{id: 'paraspacedecrease', img: 'paraspacedecrease', text: _UNO('.uno:ParaspaceDecrease'), uno: 'ParaspaceDecrease'}
```

Comment:
this should be replaced with the correspondent UNO commands for increasing and decreasing character (which I don't seem to find)

Reply:
couldn't find them in https://wiki.documentfoundation.org/Development/DispatchCommands nor searching in core. @eszkadev pointed that `.uno:Spacing?Spacing:short=60` should work. 

It didn't, probably because I should do something else?